### PR TITLE
Fix some clippy warnings

### PR DESCRIPTION
**Pull request by remilauzier**
## Description
Fix clippy warning and running cargo fmt

## Motivation and Context
Augment the quality of the code by following clippy suggestion.

## How Has This Been Tested?
I have run cargo test and it didn't regress.

## Checklist:

- [x] Acknowledged that by making this pull request I release this code under an MIT/Apache 2.0 dual licensing scheme.
- [x] My code follows the code style of this project.
- [ ] If my change required a change to the documentation I have updated the documentation accordingly.
- [ ] I have updated the content of the book if this PR would make the book outdated.
- [ ] I have added tests to cover my changes.
- [ ] My code is used in an example.

### CODE REVIEW

File: amethyst_rendy/src/visibility.rs [hunk 1]
```diff
@@ -137,7 +137,7 @@ impl System for VisibilitySortingSystem {
                             entity_query
                                 .iter(world)
                                 .map(|(entity, transform, transparent, sphere)| {
-                                    let pos = sphere.clone().map_or(origin, |s| s.center);
+                                    let pos = sphere.map_or(origin, |s| s.center);
```
Comment (jojolepro) [inline amethyst_rendy/src/visibility.rs:140]: Is the clone here required?

**Review by jojolepro — COMMENTED** [review]
Thanks a lot! :hearts:

I just have a small question about a clone() that was added.

File: amethyst_rendy/src/visibility.rs (hunk 1 above)
Comment (remilauzier) [reply amethyst_rendy/src/visibility.rs:140]: I remove it since clippy said it was unnecessary. You can see the green part where i remove the clone.

Comment (jojolepro) [reply amethyst_rendy/src/visibility.rs:140]: oh.... :D 
I'm not sure what I was thinking, thanks for pointing it out x)

**Review by jojolepro — APPROVED** [review]
Thank you! :hearts:

**Pull request merged**